app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1> <%= @mechanic.name %> Info </h1>
+<h3> Years of experience: <%= @mechanic.years_experience %></h3>
+
+<h2>Rides working on</h2>
+<section class='rides'>
+  <ol>
+    <% @mechanic.rides.each do |ride| %>
+    <li>
+      <h3>Ride name: <%= ride.name %></h3>
+    </li>
+    <%end%>
+  </ol>
+</section>
+
+<h2>Add existing ride to <%=@mechanic.name%> workload</h2>
+  <section class='addride'>
+    <%=form_with model: [@ride, @mechanic], local: true do |form|%>
+    <%= form.label :ride_id, "Ride Id:" %>
+    <%= form.text_field :ride_id %>
+
+    <%= form.submit "Add ride" %>
+    <%end%>
+  </section>
```

Comment:
The only issue I see with this view is the path for the form. I think you have the variables backward. `@mechanic` should come before `@ride` since we're creating a ride for a mechanic.

My only other question is where the `@ride` variable is coming from since you did not set it up in the `show` action in the `MechanicsController` so I would imagine this value is `nil`. I would stick with simple `form_with model:` that deals with just 1 model for now and for anything else use `form_with url:` until you get more comfortable with the `form_with model:` syntax and passing multiple objects.